src/unstract/adapters/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.7.1"
+__version__ = "0.7.2"
```

Comment:
We can bundle this up with https://github.com/Zipstack/unstract-adapters/pull/29/files

Reply:
It can be resolved as we merge the PRs - in this PR I don't believe there was any breaking change. Will change it to 0.8.0 anyway

Reply:
Or we can leave it for now and merge on conflict @chandrasekharan-zipstack No prob. Just added the comment to make sure we do this.